# This is a message to send command data to an Mikrokopter.
#
# Command data range from -128 to 128 forced by Mikrokopter.
# We couldn't change resolution.
# by Inkyu

Header header

int32 pitch
int32 roll
int32 yaw
int32 throttle
